dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04225UpdateSystemFolderIdentifier.java

```diff
@@ -0,0 +1,110 @@
+package com.dotmarketing.startup.runonce;
+
+import com.dotcms.business.WrapInTransaction;
+import com.dotmarketing.common.db.DotConnect;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.portlets.folders.business.FolderAPI;
+import com.dotmarketing.startup.AbstractJDBCStartupTask;
+import com.dotmarketing.util.Logger;
+import java.sql.SQLException;
+import java.util.List;
+
+public class Task04225UpdateSystemFolderIdentifier extends AbstractJDBCStartupTask {
+
+    public static final String SYSTEM_FOLDER_IDENTIFIER = FolderAPI.SYSTEM_FOLDER_ID;
+    public static final String UPDATE_FOLDER_QUERY = "update folder set identifier = ? where inode = 'SYSTEM_FOLDER'";
+    public static final String UPDATE_IDENTIFIER_QUERY = "update identifier set id = ? where asset_name = 'system folder'";
+    public static final String DROP_CONSTRAINT_QUERY = "ALTER TABLE Folder drop constraint folder_identifier_fk";
+    public static final String MYSQL_DROP_CONSTRAINT_QUERY = "ALTER TABLE Folder DROP FOREIGN KEY folder_identifier_fk";
+    public static final String CREATE_CONSTRAINT_QUERY = "ALTER TABLE Folder add constraint folder_identifier_fk foreign key (identifier) references identifier(id)";
+
+    @Override
+    public boolean forceRun() {
+        return true;
+    }
+
+    @Override
+    public void executeUpgrade() throws DotDataException {
+        DotConnect dc = new DotConnect();
+
+        //Drop folder_identifier_fk
+        try {
+            DbConnectionFactory.getConnection().setAutoCommit(true);
+            if (DbConnectionFactory.isMySql()) {
+                dc.setSQL(MYSQL_DROP_CONSTRAINT_QUERY);
+            } else {
+                dc.setSQL(DROP_CONSTRAINT_QUERY);
+            }
+            Logger.info(this, "Executing Query: " + dc.getSQL());
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "Executing Query: " 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)